policyengine_us/parameters/gov/states/id/tax/income/subtractions/aged_or_disabled/amount.yaml

```
@@ -0,0 +1,16 @@
+description: Idaho provides the following aged or disabled deduction amount for members of the taxpayers household.
+values:
+  1999-01-01: 1_000
+metadata:
+  reference:
+    - title: Idaho Title 63, Chapter 30, Section 63-3022E (4)
+      href: https://legislature.idaho.gov/statutesrules/idstat/title63/t63ch30/sect63-3022e/ 
+    - title: Idaho Form 39R, Resident Supplemental Schedule and Instructions 2022
+      href: https://tax.idaho.gov/wp-content/uploads/forms/EFO00088/EFO00088_12-30-2022.pdf#page=9
+    - title: 2021 Idaho Code, Title 63, Chapter 30, Section 63-3022E (1)
+      href: https://law.justia.com/codes/idaho/2021/title-63/chapter-30/section-63-3022e/
+    - title: Form 39R, Resident Supplemental Schedule 2021 approved
+      href: https://tax.idaho.gov/wp-content/uploads/forms/EFO00088/EFO00088_09-23-2021.pdf#page=1
+  unit: currency-USD
+  label: Idaho development disabilities deduction amount
```

similarly apply to the credit
```suggestion
  label: Idaho aged or disabled deduction
```